src/main/java/Todo.java

```diff
@@ -7,6 +7,4 @@ public String getTaskIdentity() {
         String todoSymbol = "[T]";
```

Comment:
For some messages like "[T]", regarding the coding quanlity, it is better to be put in an enum class or in a final string list.